src/driver-marsgaming/marsgaming-leds.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include "stdint.h"
+#include "libratbag.h"
+
+enum marsgaming_led_brightness_intensity {
+	MARSGAMING_LED_BRIGHTNESS_OFF = 0x00,
+	MARSGAMING_LED_BRIGHTNESS_LOW = 0x01,
+	MARSGAMING_LED_BRIGHTNESS_MEDIUM = 0x02,
+	MARSGAMING_LED_BRIGHTNESS_HIGH = 0x03
+};
+
+#define MARSGAMING_LED_BREATHING_OFF 0
+
+struct marsgaming_led_color {
+	uint8_t red; // 0xff - red
+	uint8_t green; // 0xff - green
+	uint8_t blue; // 0xff - blue
```

Comment:
These comments are confusing, they seem like description of the value, like, when it's 0xff, the LED is bright, but if I understand correctly, it's the other way around.

Reply:
Your understanding is correct. I will modify the comments to make it clearer.